### ISSUE TITLE
`OMP_TUNING.ExecuteBidirectional` c++ unit test fails

### ISSUE THREAD

**Issue by cconvey** [NervanaSystems/ngraph-mxnet#246]
On our `master` branch (currently commit ad81ff24ab3b0fd23abd77a08a6d72431ecd9a31), the following command fails:
```
GTEST_FILTER=` make runtest
```

I get that error both with and without the following environment variables being defined:
* `OMP_NUM_THREADS=56`
* `KMP_AFFINITY=granularity=fine,compact,1,0`

This may be a regression, considering that the first time I encountered this was with a test command that used to succeed:
```
GTEST_FILTER=-*_PERF.*:-*Timing* make runtest
```

One reason this might _not_ be a regression is if the test `OMP_TUNING.ExecuteBidirectional` was added to the MXnet 1.2 development branch, and we never tried running the c++ unit tests after merging that branch to our `master`.

**Comment by cconvey** [comment]
Note: @louisfeng says that he does _not_ encounter this error.  

In his case, he does _not_ have `OMP_NUM_THREADS` or `KMP_AFFINITY` set.

**Comment by cconvey** [comment]
I think I've narrowed down my problem: As usual, when building the bridge code I set a number of environment variables to control the Make details.  E.g., `make DEBUG=1 USE_CUDA=0 USE_NGRAPH=1 ...`.

When I ran `make runtest`, I wasn't setting those variables.  Given that `make runtest` can cause some additional compiling and linking, that seems to have introduced some subtle bug that surfaces with the `OMP_TUNING.ExecuteBidirectional` test.

The workaround is to just use the same environment variables during both steps.

Since this is an abnormal use of the build system, I'm going to close this Bug as "wontfix".
